client/templates/includes/gillespyResultsEnsemble.pug

```diff
@@ -66,13 +66,13 @@ div#workflow-results-ensemble.card.card-body
       div
 
         h5.inline Plot Trajectory Mean
-        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajmean", data-hook="collapse-trajmean") +
+        button.btn.btn-outline-collapse(data-toggle="collapse", data-target="#collapse-trajmean", id="collapse-trajmean", data-hook="collapse-trajmean") +
```

Comment:
The element on line 61 already has this id value I would suggest adding '-btn' to the end of the existing id value.

No element use my suggested id value